moduleroot/spec/spec_helper.rb.erb

```diff
@@ -43,6 +43,11 @@ RSpec.configure do |c|
     Puppet.settings[:strict] = <%= @configs['strict_level'] %>
   end
   <%- end -%>
+  c.after(:suite) do
+    <%- if @configs['enable_coverage_reports'] -%>
```

Comment:
I think the conditional should wrap the entire `c.after(:suite)` block, so that it's not an empty block if `coverage_report` is `false`.

Reply:
I put it inside the block since an empty block shouldn't hurt anything and this sets the file up for others adding additional things to this block. If it wraps the entire thing then the next contributor would have to adjust this code too. If you feel strongly about it I will change it though.

Reply:
Mostly just a stylistic nit, since the rendered file could potentially have an empty block, which isn't a big deal. No change necessary.